### PR TITLE
FIX: Require the stable branch of core modules

### PR DESCRIPTION
**Pull request by NightJar**
Update the composer.json and Readme to reflect a more accurate set of requirements

Fixes my accidental merge with outstanding change requests in #47

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
         }
     ],
     "require": {
-        "silverstripe/framework": "^4@dev",
-        "silverstripe/admin": "^1@dev"
+        "silverstripe/framework": "^4",
+        "silverstripe/admin": "^1"
```
Comment (robbieaverill) [inline composer.json]: Dan’s request was to make the constraint specific here too, so `^1.0.2` would be good!

Comment (NightJar) [reply composer.json]: Yep, realised it as I committed 😅

**Comment by robbieaverill** [comment]
Scrutinizer config fixed with https://github.com/silverstripe/silverstripe-taxonomy/commit/270b08f939424781ccda6e3f3a57390a028453a4

**Comment by robbieaverill** [comment]
I think in future it'd be good to have less commits than changes (i.e. squash them and use a sensible commit message) =)

**Pull request merged**